src/2sv/key/Touch.vue

```diff
@@ -78,29 +78,19 @@ export default {
       }
     },
     async create() {
-      if (this.isSupported) {
-        this.newU2f = await add('u2f')
-      }
-  
-      const TEN_SECS = 10000
-      absTimeout && clearTimeout(absTimeout)
-      absTimeout = setTimeout(() => {
+      if (!browserSupportsWebauthn()) {
```

Comment:
this is bound to be 100% better, excellent work here 🌟

Reply:
yeah, a first class credential api and not a polyfill hack is a nice change